CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-This release deprecates the use of gcm as an app type and refactors Sygnal to use fcm as an app type exclusively. Due to these changes, some prometheus metrics names were updated to refer to Firebase Cloud Messaging (fcm) instead of the legacy Google Cloud Messaging (GCM) services. You can find a list of the changed metric names below:
+This release deprecates the use of 'gcm' as an app type and refactors Sygnal to use 'fcm' as an app type exclusively. Due to these changes, some Prometheus metrics' names were updated to refer to Firebase Cloud Messaging (FCM) instead of the legacy Google Cloud Messaging (GCM) services. You can find a list of the changed metric names below:
```

Comment:
```suggestion
This release deprecates the use of `gcm` as an app type and refactors Sygnal to use `fcm` as an app type exclusively. Due to these changes, some Prometheus metrics' names were updated to refer to Firebase Cloud Messaging (FCM) instead of the legacy Google Cloud Messaging (GCM) services. You can find a list of the changed metric names below:
```

(I would probably have said to use \` as it will show it in code blocks, since it's a literal from the config file)

Reply:
From my experience, `gcm` is what I would use if `gcm` were the name of a config option key. However, I would write "gcm" if I were referring to a config option *value*.

Both are fine though, I don't think it needs to be debated too much.